config/samples/xgboost-dist/xgboostjob_v1alpha1_iris_train_gcp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: "xgboostjob.kubeflow.org/v1alpha1"
+kind: "XGBoostJob"
+metadata:
+  name: "xgboost-dist-iris-test-train-gcp"
+spec:
+  xgbReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: Never
+      template:
+        apiVersion: v1
+        kind: Pod
+        spec:
+          containers:
+          - name: xgboostjob
+            image: docker.io/merlintang/xgboost-dist-iris:1.1
```

Comment:
the image name is not correct, you need to build the new image withe new code.

Reply:
for sure, thanks for your advice

Reply:
just double check, you mean build a new image with new python code and update the new image to all yaml files in this folder.
Do I understand correct?